sim-cli/src/main.rs

```diff
@@ -24,7 +24,12 @@ async fn main() -> anyhow::Result<()> {
 
     for node in nodes {
         let lnd = LndNode::new(node.address, node.macaroon, node.cert).await?;
-        clients.insert(node.id, Arc::new(lnd));
+
+        let node_info = lnd.get_info().await?;
+        println!("Node info {:?}", node_info);
+
+        // TODO: validate node has keysend feature
+        clients.insert(node_info.pubkey, Arc::new(lnd));
```

Comment:
We only need this for destinations so I'm not sure it needs to live here?

Reply:
move to validation